BrowserKit/Sources/TabDataStore/TabDataStore.swift

```diff
@@ -54,17 +54,21 @@ public actor DefaultTabDataStore: TabDataStore {
             guard let fileURL = windowURLPath(for: uuid, isBackup: false),
                   fileManager.fileExists(atPath: fileURL),
                   let windowData = parseWindowDataFile(fromURL: fileURL) else {
-                logger.log("Failed to open window/tab data for UUID: \(uuid)", level: .fatal, category: .tabs)
+                logger.log("Failed to open window/tab data for UUID: \(uuid)", level: .warning, category: .tabs)
```

Comment:
This can remain as a fatal. The backup isn't perfect because it may be missing some data so it's still useful for us to be informed when the first attempt is failing in large numbers.

Reply:
For both this one and the backup error it would be useful to track which guard check failed. Is it the fileURL, does the file exist or does it fail to parse.

Reply:
Ok, we can change this back!

Could you let me know where I can go in Sentry to see events that are logged after a `fatal` like this? I thought part of our problem tracking down the issue this morning was because we couldn't see the logs after the fatal. 🤔  It would be cool if we can search for a log in a specific user's session and see everything coming before/after.

**RE extra checks:**
We noticed that the fail to parse case does have it's own warning in TabFileManager.swift:
```
logger.log("Error decoding window data: \(error)",
                       level: .debug,
                       category: .tabs)
```
So I think that covers the parse case, if that sounds ok. I'll add a couple more checks here to log for the fileURL and file exists cases.